python/py-pytest-subtests/Portfile

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-pytest-subtests
+version             0.4.0
+revision            0
+categories-append   devel
+platforms           darwin
+supported_archs     noarch
+license             MIT
+
+python.versions     36 37 38 39
+
+maintainers         nomaintainer
+
+description         unittest subTest() support and subtests fixture.
+
+long_description    {*}${description}
+
+checksums           rmd160  f372969f3e9c4e759419236ea854785f26a11d30 \
+                    sha256  8d9e2c1d1dce11f7b7d2c9d09202ebfc7757b7ff0cac9b72ad328edfe7ee037b \
+                    size    10751
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools_scm_git_archive
+
+    depends_lib-append \
+                    port:py${python.version}-setuptools
+
+    # no tests on PyPI archive
+    # depends_test-append \
+    #                 port:py${python.version}-pytest
+
+    # test.run        yes
+    # test.cmd        py.test-${python.branch}
+    # test.target     tests
+    # test.env        PYTHONPATH=${worksrcpath}/build/lib
```

Comment:
if there are not tests on PyPI (which is fine), just remove this block instead of commenting it out

Reply:
Thanks done.